odfuzz/odfuzz.py

```diff
@@ -86,6 +87,9 @@ def run_fuzzer(bind, parsed_arguments, collection_name):
         sys.exit(1)
     except gevent.Timeout:
         signal_handler(collection_name)
+    except Exception:
+        print(traceback.format_exc())
```

Comment:
We maybe still mistunderstanding each other.  I need usage of logging together with console output.